ui/src/utils/timeSeriesTransformers.js

```diff
@@ -198,6 +198,19 @@ export const filterTableColumns = (data, fieldNames) => {
   return filteredData[0].length ? filteredData : [[]]
 }
 
+export const orderTableColumns = (data, fieldNames) => {
+  const fieldsSortOrder = fieldNames.map(fieldName => {
+    return _.findIndex(data[0], dataLabel => {
+      return dataLabel === fieldName.internalName
+    })
+  })
+  const filteredFieldSortOrder = _.filter(fieldsSortOrder, f => f !== -1)
```

Comment:
also could use filter from fast.js instead of _

Reply:
thank you.